ci.sh

```diff
@@ -19,7 +19,7 @@ cat > /tmp/ignoretests.txt << EOF
 EOF
 
 ./shellcheck.sh
-./snc.sh
+SNC_GENERATE_MACOS_BUNDLE=0 ./snc.sh
```

Comment:
Dunno if we can achieve the downgrade using public repos (assuming this runs on rhel9).